radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -255,21 +264,12 @@ public RadixAddress getMyAddress() {
 		return universe.getAddressFrom(identity.getPublicKey());
 	}
 
-	public Observable<Data> getData(RadixAddress address) {
```

Comment:
Should we perhaps change "Data" to "Message"? Similarly, "UnencryptedData" to "UnencryptedMessage" etc.

Reply:
Good idea